projects/valtimo/form-link/src/lib/form-flow.service.ts

```diff
@@ -17,22 +17,28 @@
 import {Injectable} from '@angular/core';
 import {HttpClient} from '@angular/common/http';
 import {Observable} from 'rxjs';
-import {FormFlowDefinition} from './models';
+import {FormFlowDefinition, TaskProcessLinkResult} from './models';
 import {ConfigService} from '@valtimo/config';
 
 @Injectable({
   providedIn: 'root',
 })
 export class FormFlowService {
-  private valtimoApiConfig: any;
+  private valtimoEndpointUri!: string;
 
   constructor(private http: HttpClient, private configService: ConfigService) {
-    this.valtimoApiConfig = configService.config.valtimoApi;
+    this.valtimoEndpointUri = configService.config.valtimoApi.endpointUri;
   }
 
   getFormFlowDefinitions(): Observable<FormFlowDefinition[]> {
     return this.http.get<FormFlowDefinition[]>(
-      `${this.valtimoApiConfig.endpointUri}process-link/form-flow-definition`
+      `${this.valtimoEndpointUri}process-link/form-flow-definition`
+    );
+  }
+
+  getTaskProcessLink(taskId: string): Observable<TaskProcessLinkResult> {
```

Comment:
Not sure this should be in the form flow service. Form flow is only 1 of the possible implementations

Reply:
Which service would you deem more appropriate? FormLinkService? TaskService? I would go with the latter personally